blocks/header/header.css

```diff
@@ -300,5 +365,65 @@ header {
 
 /* tools */
 .nav-tools {
+  --nav-tools-gap: 12px;
+  --nav-tools-padding-block: 20px;
+  
+  display: none;
   grid-area: tools;
+  padding-block: var(--nav-tools-padding-block);
+  
+  @media (min-width: 768px) {
+    --nav-tools-padding-block: 0;
+
+    display: block;
+  }
+  
+  .default-content-wrapper {
+    display: flex;
+    flex-direction: column;
+    gap: var(--nav-tools-gap);
+    
+    @media (min-width: 768px) {
+      --nav-tools-gap: 20px;
+
+      flex-direction: row;
+      justify-content: flex-end;
+      gap: 20px;
+    }
+  }
+
+  a.button:any-link, 
+  button {
+    margin: 0;
+
+    @media (max-width: 768px) {
+      min-height: auto;
+      padding: 0;
+      border: 0;
+      background: none;
+      font-size: var(--txt159-font-size);
+      line-height: var(--txt159-line-height);
+      font-weight: 400;
```

Comment:
not needed

Reply:
Actually this is needed, as the buttons are having a 700 weight by default.